ostree/ostree_dest.go

```diff
@@ -35,9 +48,13 @@ type descriptor struct {
 	Digest digest.Digest `json:"digest"`
 }
 
+type fsLayersSchema1 struct {
+	BlobSum digest.Digest `json:"blobSum"`
+}
+
 type manifestSchema struct {
-	ConfigDescriptor  descriptor   `json:"config"`
-	LayersDescriptors []descriptor `json:"layers"`
+	LayersDescriptors []descriptor      `json:"layers"`
+	FSLayers          []fsLayersSchema1 `json:"fsLayers"`
```

Comment:
It would be cleaner to use the future `c/image/manifest` subpackage to parse the manifest… but it’s doesn’t break any other transports, so non-blocking I guess.